src/libraries/System.Runtime/tests/System.IO.Tests/BinaryReader/BinaryReaderTests.cs

```diff
@@ -438,6 +444,31 @@ public void Read_CharSpan_ThrowIfDisposed()
             }
         }
 
+        [Theory]
+        [InlineData(100, 10)]
+        [InlineData(10, 10)]
+        [InlineData(10, 0)]
+        [InlineData(0, 0)]
+        public void ReadExactly_ByteSpan(int sourceSize, int destinationSize)
+        {
+            using (var stream = CreateStream())
+            {
+                var source = new byte[sourceSize].AsSpan();
```

Comment:
nit: `AsSpan` is needed only for test projects that target older monikers like Full .NET Framework (because `Span<T>` is not part of `mscorlib.dll` in Full Framework, so we could not have added an implicit cast operator from array to span for Full Framework).

```suggestion
                byte[] source = new byte[sourceSize];
```

Reply:
I see, thanks for the explanation. I have updated the tests to remove the `.AsSpan()` calls.